content/en/people/index.adoc

```diff
@@ -0,0 +1,27 @@
+---
+title: "People"
+description: "ACL Special Interest Group on Turkic Languages People"
+date: 2020-08-27T19:25:12+02:00
+lastmod: 2020-08-27T19:25:12+02:00
```

Comment:
I'm not really familiar with where this metadata is used, but should it say 2022?

Reply:
I will remove these, leftover